partner/pms-common/src/main/java/io/mosip/pms/common/util/AuthenticationContextRefUtil.java

```diff
@@ -86,8 +98,11 @@ private Map<List<String>, String> getAllClaims() {
 					new TypeReference<Map<String,Claims >>() {
 					});
 			Map<List<String>, String> claimsMap = new HashMap<>();
+			List<String> allowedClaims = getSupportedClaims();
 			for(Map.Entry<String,Claims> mapElement : map.entrySet()) {
-				claimsMap.put(convertStringToList(mapElement.getValue().getAttributeName()),mapElement.getKey());
+				if(allowedClaims.contains(mapElement.getKey())) {
+					claimsMap.put(convertStringToList(mapElement.getValue().getAttributeName()),mapElement.getKey());
+				}
```

Comment:
I see this method is returning "null" instead can we throw an exception?
With this, we don't have to worry about null checks in all the callers.

Reply:
Done the same as suggested